integration/tests/iam_v2_no_legacy_to_v2_force_upgrade.sh

```diff
@@ -15,7 +15,7 @@ test_upgrade_strategy="none"
 
 # a2-iam-no-legacy-integration verifies permissions on an IAM v2 system 
 # without v1 legacy policies
-test_deploy_inspec_profiles=(a2-iam-no-legacy-integration)
+test_deploy_inspec_profiles=(a2-deploy-smoke)
```

Comment:
expected functionality for v2 has changed, so the `a2-iam-no-legacy-integration` won't pass on both the pre and post upgrade. Smoke tests ensure the pre-upgrade is running.